src/NLog/LayoutRenderers/GdcLayoutRenderer.cs

```diff
@@ -66,10 +66,20 @@ public class GdcLayoutRenderer : LayoutRenderer
         /// <param name="logEvent">Logging event.</param>
         protected override void Append(StringBuilder builder, LogEventInfo logEvent)
         {
-            //don't use GlobalDiagnosticsContext.Get to ensure we are not locking the Factory (indirect by LogManager.Configuration).
-            var value = GlobalDiagnosticsContext.GetObject(Item);
+            object value = GetValue();
             var formatProvider = GetFormatProvider(logEvent, null);
             builder.AppendFormattedValue(value, Format, formatProvider);
         }
+
+        /// <summary>
+        /// Get raw value.
+        /// </summary>
+        object IRawValue.GetRawValue(LogEventInfo logEvent) => GetValue();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename parameter 'logEvent' to 'logEventInfo' to match the interface declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS927)